### DISCUSSION TITLE
An error appeared during calculation

### DISCUSSION THREAD

**Discussion by Mangwuha** [Anura3D/Anura3D_OpenSource#158] [Q&A]
Hi,everyone!
I am a student. I have a problem and I need help. I wanted to complete a simple Landslide model(After turning on the contact algorithm it will break normally and there will be a slide to the right), but It reported errors during the calculations (the screenshots of building the model and reporting the errors are below). Before this, I tried to repeat the Sliding Blocks example in the tutorial manual, I followed the steps in the tutorial, but the calculation also reported errors. Both types of reporting errors are the same, I tried to find and fix the problem by myself, but unfortunately I couldn't find the problem. I hope someone can help me to figure out where am i wrong.
Thanks!

![1](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/bb452e95-fbf9-4488-ac08-c631fb608efe)
![2](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/7112b60a-82e2-4466-8b04-ab0aa117d9c3)
![bianjietiaojian ](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/fc16f494-c90a-45b5-a720-96d20b61ddf1)
![contact](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/3547fb45-989c-48fc-a4dc-b0ca7c691d20)
![Mesh](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/faa32270-b236-468d-85c3-e7b513008be9)
![calculate-setting](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/9b735560-52f2-4b70-96d4-0129ad41efc9)
![error](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/ea287616-497c-4895-ab60-e3b82007b8b6)

My goal is to achieve the results given in the figure below：
![3](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/1a3d9f55-1710-41f4-ac04-11d3dd988854)

**Comment by genclub33** [comment]
Hi!
It seems that you apply a contact between base and soil, but you don't select it an "Apply contact" in the calculation data.

**Reply by Mangwuha** [reply]
Hi!
Shouldn't the first stage be initialisation of stresses by quasi-static gravity loading？”Apply contact"is supposed to be selected in the second stage.

**Comment by genclub33** [comment]
Oh, sorry!
You're correct. It's my mistake!

**Comment by FrancescaCeccato** [comment]
This type of error usually appears if the input written in the GOM file are wrong. Usually it can be solved by remaking carefully the entire model.
The sliding block benchmark should work perfectly because we tested it several times. Maybe try with that again first. it is possible that you have something wrong int he definition of contact.
Good luck!
